test/e2e/kubectl.go

```diff
@@ -298,7 +298,11 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 
 			// Build the static kubectl
 			By("Finding a static kubectl for upload")
-			testStaticKubectlPath, err := findBinary("kubectl", "linux/386")
+			targetBinary := "linux/386"
```

Comment:
What do you mean the work with images?

I am not sure why the original test code uses linux/386. Moreover, I am not pretty sure where this test code is used, only in GCE, or other cloud platforms? So I did not change it to amd64 for all conditions. But I am pretty clear about GCI and pretty sure where do we run this test case for GCI. So I only change the behavior for GCI. If you know using amd64 is safe for all other images and cloud platforms, I am very happy to see we just use amd64.